testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/common/HadoopKerberos.java

```diff
@@ -76,6 +76,7 @@ public void extendEnvironment(Environment.Builder builder)
         });
         builder.configureContainer(TESTS, container -> {
             container.setDockerImageName(dockerImageName);
+            container.withCopyFileToContainer(forHostPath(configDir.getPath("krb5_client.conf")), "/etc/krb5.conf");
```

Comment:
Can we move this as a preparatory commit - would help us in debugging in case of flaky tests.

Reply:
moved